pkgs/desktops/deepin/dde-file-manager/default.nix

```diff
@@ -0,0 +1,235 @@
+      dde-file-manager/dde-file-manager-xdg-autostart.desktop \
+      dde-file-manager/dde-file-manager.desktop \
+      dde-file-manager/dde-file-manager.pro \
+      dde-file-manager/mips/dde-file-manager-autostart.desktop \
+      dde-file-manager/mips/dde-file-manager.desktop \
+      dde-file-manager/pkexec/com.deepin.pkexec.dde-file-manager.policy \
+      usb-device-formatter/pkexec/com.deepin.pkexec.usb-device-formatter.policy \
+      usb-device-formatter/usb-device-formatter.desktop \
+      usb-device-formatter/usb-device-formatter.pro
+      fixPath $out /etc \
+      dde-file-manager/dde-file-manager.pro \
+      dde-file-manager-daemon/dde-file-manager-daemon.pro
+
+    fixPath $out /usr \
+      common/common.pri \
+      dde-desktop/dbus/filedialog/filedialog.pri \
+      dde-desktop/dbus/filemanager1/filemanager1.pri \
+      dde-desktop/development.pri \
+      dde-dock-plugins/disk-mount/disk-mount.pro \
+      dde-file-manager-daemon/dde-file-manager-daemon.pro \
+      usb-device-formatter/usb-device-formatter.pro
+
+    sed -i -e "s,Exec=dde-file-manager,Exec=$out/bin/dde-file-manager," \
+      dde-file-manager/dde-file-manager.desktop
+
+    sed -i -e "s,Exec=gio,Exec=${glib.bin}/bin/gio," \
+      dde-desktop/data/applications/dde-trash.desktop \
+      dde-desktop/data/applications/dde-computer.desktop
+
+    sed -i -e "s,/usr/lib/gvfs/gvfsd,${gvfs}/libexec/gvfsd," \
+      dde-file-manager-lib/gvfs/networkmanager.cpp
+
+    # I do not know yet how to fix the ocurrence of QString("/etc/xdg/%1/%2") in 
+    #   dde-file-manager-lib/interfaces/dfmsettings.cpp
+    # Should it be fixed somehow, or can it stay as it is?
+
+    # I do not know yet how to deal with:
+    #   dde-file-manager-lib/sw_label/llsdeepinlabellibrary.h:        return "/usr/lib/sw_64-linux-gnu/dde-file-manager/libllsdeeplabel.so";
+    #   dde-file-manager-lib/sw_label/filemanagerlibrary.h:        return "/usr/lib/sw_64-linux-gnu/dde-file-manager/libfilemanager.so";
+    #   dde-file-manager-lib/sw_label/libinstall.sh:mkdir /usr/lib/sw_64-linux-gnu/dde-file-manager
+    #   dde-file-manager-lib/sw_label/libinstall.sh:cp libfilemanager.so libllsdeeplabel.so /usr/lib/sw_64-linux-gnu/dde-file-manager
+    # They are not present on my installations of Deepin Linux, Arch Linux and Ubuntu. Can they be ignored?
+  '';
+
+  qmakeFlags = [
+    "QMAKE_CFLAGS_ISYSTEM="
+
+    # Disable ffmpeg
+    "CONFIG+=DISABLE_FFMPEG"
+
+    # For now disable deepin-anything module, a kernel module for
+    # vfs_change monitoring to provide fast file indexing feature
+    "CONFIG+=DISABLE_ANYTHING"
+  ];
+
+  preBuild = ''
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${zlib}/lib";
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${libX11}/lib";
+  '';
+
+  postFixup = ''
+    # debuging
+    unset LD_LIBRARY_PATH
+    searchForUnresolvedDLL $out
+    searchHardCodedPaths -a $out
+  '';
+
+  #enableParallelBuilding = false;
```

Comment:
needed?

Reply:
Not really. I have been setting `enableParallelBuilding` to false in order to have intelligible error messages.